packages/vue/src/components/primitives/styles.css

```diff
@@ -10,22 +10,5 @@ html {
 }
 
 [data-amplify-spacer] {
-  @apply flex-1;
-}
-
-[data-amplify-lostcode] {
-  @apply flex flex-col items-start;
-}
-
-[data-amplify-select] {
-  @apply block w-full  rounded shadow-sm;
-  border-width: 1px;
-  padding: 0.5rem 0.75rem;
-  font-size: 1rem;
-  line-height: 1.5rem;
-  width: 97%;
-}
-
-[data-amplify-alias-label] {
-  @apply flex flex-col;
+  flex: 1;
```

Comment:
`data-amplify-spacer` is the only one that is still in use, verified that the others are vestigial.